Sources/xcprojectlint-package/NoDanglingSourceFiles.swift

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2018 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import Foundation
+
+public func noDanglingSourceFiles(_ project: Project, errorReporter: ErrorReporter) -> Int32 {
```

Comment:
`checkforDanglingSourceFiles`
or
`checkForOrphanedSourceFiles`

i've always referred to them as `orphans`, maybe it's not quite correct in this case?

Reply:
I'll stick with docker style here